pytorchfi/errormodels.py

```diff
@@ -226,28 +206,33 @@ def _flip_bit_signed(self, orig_value, max_value, bit_pos):
         else:
             bits_new[bit_loc] = "0"
         bits_str_new = "".join(bits_new)
+        logging.info("bits", bits_str_new)
 
         # GPU contention causes a weird bug...
         if not bits_str_new.isdigit():
-            print("orig value:", orig_value)
-            print("max value:", max_value)
-            print("bit_pos:", bit_pos)
-            print("total bits:", total_bits)
-            print("bit string:", bits_str_new)
+            logging.info("orig value:", orig_value)
+            logging.info("max value:", max_value)
+            logging.info("bit_pos:", bit_pos)
+            logging.info("total bits:", total_bits)
+            logging.info("bit string:", bits_str_new)
```

Comment:
Not totally sure if logging is necessary here - it should only have high level functions printed to console so that the user can easily trace whats going on, especially when running many things. Otherwise too much granularity will make difficult debugging.

Reply:
Simplified.